components/sidebar/sidebar_category/sidebar_category_sorting_menu/sidebar_category_sorting_menu.tsx

```diff
@@ -39,6 +44,10 @@ type Props = OwnProps & PropsFromRedux;
 const SidebarCategorySortingMenu = (props: Props) => {
     const [openUp, setOpenUp] = useState(false);
 
+    const activeElement = document.activeElement as HTMLElement;
```

Comment:
Same feedback here regarding `activeElement` changing unexpectedly and the callback passed to the menu item being wrapped in `useCallback`